packages/graph/hash_graph/lib/graph/src/store/postgres/query/compile.rs

```diff
@@ -39,7 +39,11 @@ impl<'c, 'p: 'c, T: PostgresQueryRecord + 'static> SelectCompiler<'c, 'p, T> {
                 with: WithExpression::default(),
                 distinct: Vec::new(),
                 selects: Vec::new(),
-                from: T::base_table(),
+                from: T::base_table().aliased(Alias {
```

Comment:
Does get a little repetitive with the alias struct. Could we do with a constructor `fn new`?

Reply:
I chose to not use a constructor here as it's hard to tell, which parameter means what as all three numbers have the same type.

Reply:
I agree, that's a good point :)